app_flutter/test/widgets/commit_box_test.dart

```diff
@@ -12,97 +11,93 @@ import 'package:cocoon_service/protos.dart' show Commit;
 import 'package:app_flutter/widgets/commit_box.dart';
 
 void main() {
-  group('CommitBox', () {
-    final Commit expectedCommit = Commit()
-      ..author = 'AuthoryMcAuthor Face'
-      ..authorAvatarUrl = 'https://avatars2.githubusercontent.com/u/2148558?v=4'
-      ..repository = 'flutter/cocoon'
-      ..sha = 'ShaShankRedemption';
-
-    testWidgets('shows information correctly', (WidgetTester tester) async {
-      await tester.pumpWidget(Directionality(
-        child: CommitBox(
-          commit: expectedCommit,
+  final Commit expectedCommit = Commit()
+    ..author = 'AuthoryMcAuthor Face'
+    ..authorAvatarUrl = 'https://avatars2.githubusercontent.com/u/2148558?v=4'
+    ..repository = 'flutter/cocoon'
+    ..sha = 'ShaShankRedemption';
+  final String shortSha = expectedCommit.sha.substring(0, 7);
+  final Widget basicApp = MaterialApp(
+    home: Material(
+      child: Center(
+        child: SizedBox(
+          height: 100.0,
+          width: 100.0,
+          child: CommitBox(
+            commit: expectedCommit,
+          ),
         ),
-        textDirection: TextDirection.ltr,
-      ));
+      ),
+    ),
+  );
 
-      expect(find.byType(CachedNetworkImage), findsOneWidget);
-    });
+  testWidgets('CommitBox shows information correctly', (WidgetTester tester) async {
+    await tester.pumpWidget(basicApp);
+    await expectLater(find.byType(Overlay), matchesGoldenFile('commit_box_test.idle.png'));
```

Comment:
Thanks for adding golden testing!

Reply:
/cc @Piinks it's great to see how easy it is use! 🎉

Reply:
Sweet! Does fluter/cocoon already have golden file tests?

Reply:
Quick search, it looks like this may be the first golden file test in the repo. Do we want to check in the image to the repository? This repo isn't currently set up to use another service.

Reply:
Should be fine to put it in here, we only avoid putting it in the framework repo because we're trying to avoid binaries in the repo people clone when installing Flutter.